lab-padget/test/readfile-test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+// npm run test
+console.log('evaluating readfile.js');
+// it should read three files and log the first 8 bytes in hex.
+
+const expect = require('chai').expect;
+const Buffer = require('buffer').Buffer;
+const fs = require('fs');
+// require readFile in test.
+
+let mock = [ '5475726e69702067', '476f646172642061', '596f752068617665' ];
+console.log(mock);
+// compare this array to out array.
```

Comment:
So nice test, but per instructions you must test the functionality of your helper function. So you actually have to require that file in.